ansible/group_vars/all.yml

```diff
@@ -31,8 +31,8 @@ stanchion_data_dir: "{{ encrypted_root }}/stanchion"
 # commcare-hq connects to an S3-compatible service, which is Riak CS
 s3_blob_db_enabled: "{{ 'true' if groups.get('riakcs') else '' }}"
 s3_blob_db_url: "http://{{ groups['proxy'][0] }}:{{ riakcs_port }}"
-s3_blob_db_access_key: "{{ hostvars[riakcs_control]['riak_key'] if s3_blob_db_enabled else '' }}"
-s3_blob_db_secret_key: "{{ hostvars[riakcs_control]['riak_secret'] if s3_blob_db_enabled else '' }}"
+s3_blob_db_access_key: "{{ hostvars[riakcs_control].get('riak_key', 'not-set') if s3_blob_db_enabled else '' }}"
+s3_blob_db_secret_key: "{{ hostvars[riakcs_control].get('riak_secret', 'not-set') if s3_blob_db_enabled else '' }}"
```

Comment:
I don't think this is right. The `riak_key` and `riak_secret` variables should be set any time that these variables are accessed and `s3_blob_db_enabled` evaluates to true. If they're not set then it seems like something is wrong with the environment. Can you give me reproduction steps for the problem that caused you to write this?